src/Sentry.Unity.Editor.iOS/SentryXcodeProject.cs

```diff
@@ -22,22 +22,21 @@ internal class SentryXcodeProject : IDisposable
         private readonly string _mainPath = Path.Combine("MainApp", "main.mm");
         private readonly string _optionsPath = Path.Combine("MainApp", OptionsName);
         private readonly string _uploadScript = @"
-
 process_upload_symbols() {{
-    ./{0} --log-level=info upload-dif {1} $BUILT_PRODUCTS_DIR 2>&1 | tee ./sentry-symbols-upload.log
+    ./{0} upload-dif --force-foreground {1} $BUILT_PRODUCTS_DIR 2>&1 | tee ./sentry-symbols-upload.log
```

Comment:
why the removal of `--log-level=info`? is it defined elsewhere?

Reply:
Yes. We write the `log-level` into `sentry.properties`. Just like on Android.